File: android-mobile/app/src/main/java/ng/emedic/emedic_mobile/networking/services/LoginService.java
package ng.emedic.emedic_mobile.networking.services;

import ng.emedic.emedic_mobile.networking.Routes;
import ng.emedic.emedic_mobile.networking.models.Login;
import ng.emedic.emedic_mobile.networking.response.AuthResponse;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface LoginService {
    @POST(Routes.LOGIN_END_POINT)
    Call<AuthResponse> login(@Body Login login);
}

File: bookit.ng/app/src/main/java/com/bookitng/bookitng/networking/services/LoginService.kt
package com.bookitng.bookitng.networking.services

import com.bookitng.bookitng.models.LoginModel
import com.bookitng.bookitng.models.LoginPayload
import com.bookitng.bookitng.networking.Routes
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface LoginService {

    @POST(Routes.LOGIN_ENDPOINT)
    fun initiateLogin(@Body loginModel: LoginModel): Call<LoginPayload>
}